README.md

```diff
@@ -51,12 +51,9 @@
 
 -Client
 
-- **This Project has a Readme.md file**
-- **This Project has a gitignore file**
-- **This projects follows all Microverse Linters**
-- **This project takes user inputs of Book title and Authors**
-- **This project displays the input for Book details dynamically on the DOM**
-- **This project saves the Book details insode the Browser**
+- **HTML**
+- **CSS**
+- **JavaScript**
 
```

Comment:
- [ ] As suggested by the previous reviewer, kindly define a header "Key features" where you can list what people can "do" with your project, e.g. Create a book, See a list of books and Remove books.